.github/workflows/server.yml

```diff
@@ -45,7 +47,6 @@ jobs:
           MODE: gitlab
           LOG_LEVEL: debug
           APOLLO_PATH: /lib
```

Comment:
is apollo being used, if not, this variable might have to be replaced. A possible name could be "BASE_PATH"

Reply:
Apollo server is still responsible for setting running the RAMS in conjunction with nestjs. It is just not apollo client taht i am utilizing for querying requests to the Gitlab API